scripts/ci/install-deps

```diff
@@ -40,14 +40,9 @@ chmod +x kubectl
 $SUDO mv kubectl /usr/local/bin/
 
 if [[ "$DISTRO_TYPE" == "$DISTRO_TYPE_UPSTREAM" ]]; then
-  # minikube and crictl dependency
-  echo "Installing minikube and crictl"
-  curl -Lo crictl.tar.gz "https://github.com/kubernetes-sigs/cri-tools/releases/download/${KUBE_VER}/crictl-${KUBE_VER}-linux-amd64.tar.gz"
-  tar -zxvf crictl.tar.gz
-  chmod +x crictl
-  $SUDO mv crictl /usr/local/bin/
-  rm crictl.tar.gz
-  curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
-  chmod +x minikube
-  $SUDO mv minikube /usr/local/bin/
+  # kind
+  echo "Installing kind"
+  curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VER}/kind-$(uname)-amd64
```

Comment:
If you want use this you need specify exact version with this url. If you want use latest it should be `https://github.com/kubernetes-sigs/kind/releases/latest/download/kind-$(uname)-amd64` (latest if before donwload in uri in this case)

Reply:
@ssimk0, I understand what are you saying. Original link is not working due to KIND_VER=latest. So it should be changed to a specific version or there is different link to get latest one.

Reply:
Yes, I also thought `latest` would work but it's not. The default is `v0.7.0` for that reason.